src/main/res/values/strings.xml

```diff
@@ -870,7 +870,7 @@
 	<string name="pref_cache_location_title">Cache storage location</string>
 
 	<string name="error_cache_dir_does_not_exist_title">Cache directory missing</string>
-	<string name="error_cache_dir_does_not_exist_message">The cache directory could not be accessed. Please choose a different location in the cache settings.</string>
+	<string name="error_cache_dir_does_not_exist_message">Could not access cache directory. Please choose a different location in the cache settings.</string>
```

Comment:
Overly terse, I prefer not to omit "the" in error messages/etc (except in toasts, where terseness is good)

Reply:
It is consistent. It is more terse only in the interest of brevity. The original is longer and less terse.

Reply:
Longer, more grammatical sentences are often easier to read/comprehend that ones which have all grammatical niceties stripped out. In this particular case I don't mind too much either way.

Reply:
Flipping the verb, subject and objective around like this isn't a nicety. Why the cache directory has to be a marked noun I don't know. I can add the "the" again if you want.

Reply:
Okay thanks, let's go with the new version but add the "the" again